tesseract-clickhouse/src/sql/primary_agg.rs

```diff
@@ -40,7 +41,13 @@ pub fn primary_agg(
     // So just swap the primary key DimSubquery to the head
 
     let mut ext_drills: Vec<_> = drills.iter()
-        .filter(|d| d.table.name != table.name)
+        .filter(|d| {
+            if let Some(it) = &d.inline_table {
```

Comment:
Just want to note here, that you can use `if it.is_some()`, but no change necessary.